internal/build/tar.go

```diff
@@ -84,11 +85,12 @@ func tarPath(ctx context.Context, tarWriter *tar.Writer, source, dest string) er
 		if sourceIsDir {
 			// Name of file in tar should be relative to source directory...
 			header.Name = strings.TrimPrefix(path, source)
-		}
-
-		if dest != "" {
-			// ...and live inside `dest` (if given)
-			header.Name = filepath.Join(dest, header.Name)
+			if dest != "" {
+				// ...and live inside `dest` (if given)
+				header.Name = filepath.Join(dest, header.Name)
```

Comment:
hmm pretty sure it's safe to `filepath.Join("", somePath)` -- so we don't actually need to check if `dest == ""`, can just call join. see: https://play.golang.com/p/XqkOgWmQ9k0